raiden-ts/src/transfers/epics/locked.ts

```diff
@@ -181,7 +184,7 @@ function makeAndSignTransfer$(
  * @param deps - RaidenEpicDeps
  * @returns Observable of transferSigned|transferSecret|transfer.failure actions
  */
-function makeAndSignTransfer(
+function sendTransferSigned(
```

Comment:
In portuguese, would also be as you suggested, but this is done like that to use a _reverse-domain-like_ pattern: higher-level domain of meanings go first, so `send` (in contrast to `receive`), `transfer` (since this is about sending transfers, in contrast with withdraws or other things) and `signed` (the signed `LockedTransfer`, in contrast with transfers being unlocked or expired), and so on. This is more or less the same naming schema for the path-like pattern for action type tags, so I'm keeping this for now, but we can discuss later.